src/Premium/Attribution.php

```diff
@@ -130,23 +97,29 @@ public function specialThanksControl( $configs ) {
 	}
 
 	/**
-	 * Adds attribution link controls to theme customizer.
-	 *
-	 * @hook kirki/fields
+	 * Adds attribution settings
 	 *
-	 * @param array $controls [description]
+	 * @since SINCEVERSION
 	 */
-	public function addControls( $controls ) {
-		$controls = array_merge( $controls, $this->getControls() );
-
-		if ( ! get_option( 'boldgrid_reseller', false ) ) {
-			unset( $controls['reseller_control'] );
-		}
-		if ( ! $this->getLicensed() ) {
-			unset( $controls['special_thanks_control'] );
-		}
```

Comment:
This logic may need to get added back in.

Reply:
This has been corrected.